src/processor/collections/reasons.py

```diff
@@ -0,0 +1,40 @@
+""" 'Reason' objects used to describe the reasons that Entries may be deleted.
```

Comment:
Bit of an odd way of doing things, but I think it will allow us to add new 'Reasons', and other similar types of objects in future if needed.